src/sentry/auth/access.py

```diff
@@ -95,8 +100,15 @@ class Access(abc.ABC):
     # would be based on the same scopes as API access so there's clarity in
     # what things mean
 
-    sso_is_valid: bool = False
```

Comment:
I dont' see any changes to call sites or usages of these values; is this really not being used?

Reply:
Replaced by a property.

Reply:
I see, those two below.  Is `sso_state` intended as a public interface then?

Reply:
I don't have a strong opinion but just curious, since it seems there will be two ways of accessing that data.

Reply:
No strong opinion either. If I were doing it from scratch, I'd prefer to make `SsoState` the public interface. Maybe I'll make a case for replacing the boolean properties later. For now, I'll err on the side of making `_sso_state` private and we can change our minds if we want.

Reply:
👍 My only main concern is changing too many things at once.  Can focus one thing down at a time while making general interface improvements.